CHANGELOG.md

```diff
@@ -44,6 +44,16 @@ Note that 3.1.0 will be the first API stable release and interfaces in this rele
 
 ## API changes
 
+### I/O
+
+* The default fields for reading and writing alignment files were changed.
+  Instead of reading the alignment information into the `seqan3::field::alignment` it is read into the
+  `seqan3::field::cigar` (both store the same information but in different types).
+  Hence, reading and writing in one line is now possible without additional reference information:
```

Comment:
`reading and writing in one line` is misleading. Better specify: _can be implemented in a single line of source code_; or paraphrase: _With this change we allow direct streaming from one sam file into another without reference information._